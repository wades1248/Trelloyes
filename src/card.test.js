import React from 'react';
import ReactDOM from 'react-dom';
import Card from './card';
import renderer from 'react-test-renderer';
import { exportAllDeclaration } from '@babel/types';

describe('card component', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<Card title='card' content="card" />, div);
        ReactDOM.unmountComponentAtNode(div);
    });
    it('renders UI as expected', () => {
        const tree = renderer
            .create(<Card title="card" content="card" />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});