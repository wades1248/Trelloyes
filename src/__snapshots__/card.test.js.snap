// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`card component renders UI as expected 1`] = `
<div
  className="Card"
>
  <h3>
    card
  </h3>
  <p>
    card
  </p>
</div>
`;
